src/App.jsx

```diff
@@ -14,70 +15,56 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [title, setTitle] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const buttonClick = (good) => {
```

Comment:
since functions should start with a verb
```suggestion
  const handleButtonClick = (good) => {
```